wrappers/redis/redis_test.go

```diff
@@ -0,0 +1,344 @@
+package epsagonredis_test
```

Comment:
add  a test for the case the tracer AddEvent operation fails - PanicAddEvent: true,when initializing the mocked tracer. You can see an example at the fiber tests. (just to make sure the user original code isn't paniced/errored)